src/drivers/mysql2.ts

```diff
@@ -178,6 +178,11 @@ export function preamble(queries: unknown) {
             undefined,
             factory.createIdentifier("RowDataPacket")
           ),
+          factory.createImportSpecifier(
+            false,
+            undefined,
+            factory.createIdentifier("ResultSetHeader")
+          ),
```

Comment:
We should only import this if there is a `execlastid` query for the given file.

Reply:
fixed: [`8d3fcaf` (#13)](https://github.com/sqlc-dev/sqlc-gen-typescript/pull/13/commits/8d3fcaf677ef0708f8c6715bb8bbff9f67b75537)